services/user/index.js

```diff
@@ -63,7 +63,7 @@ export const findUserByEmail = async (email) => {
  */
 export const findUserById = async (id) => {
   const user=await userModel.findById({
```

Comment:
You can pass the id directly `findById(id)`, here refer this [doc](https://mongoosejs.com/docs/api/model.html#Model.findById())